DOCS/license-inclusion-principles.md

```diff
@@ -7,42 +7,41 @@ Determining whether a candidate license should be included on the SPDX License L
   * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
-* Binary-only software licenses or licenses that do not provide for the availability of the source code will not be included on the SPDX License List.
+* Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.
+* The license has identifiable and stable text; it is not in the midst of drafting. The license steward is committed to not modifying after addition to the list and to versioning new versions in the future.
   
 **Other factors (roughly in order of importance):**
-* The license substantially complies with one of the following open source definitions (even if not submitted for approval, as in the case of OSI) or these organization consider the license to meet their definition:
+1. The license substantially complies with one of the following open source definitions (even if not submitted for approval or these organization have not considered the license):
   * [Open Source Definition (OSD)](https://opensource.org/osd) from the Open Source Initiative (OSI)
   * [Free Software Definition](https://www.gnu.org/philosophy/free-sw.en.html) from the Free Software Foundation (FSF)
   * [Open Source Hardware Definition](https://www.oshwa.org/definition/) from the Open Source Hardware Association (OSHWA)
-  * [Open Definition](http://opendefinition.org/od/2.1/en/) from the Open Knowledge Foundation (for open data)
-* The license is structured to be generally usable by anyone. It is not a "vanity" license or specific to one project, consortium or corporation.
-* The license has actual, substantial use such that it is likely to be encountered. Substantial use may be demonstrated via use in many projects, or in one or a few highly significant projects.
-* The license has identifiable and stable text; it is not in the midst of drafting. The license steward is committed not modifying after addition to the list and to versioning new versions in the future.
-* If the license does not substantively comply with one of the above open source definitions, then the license is primarily intended for free distribution of content (including, in the case of software, its source code) with limited restrictions, and also meets other factors listed here.
-* The license steward supports or is at least aware of and does not oppose its submission to the SPDX License List.
+  * [Open Definition](http://opendefinition.org/od/2.1/en/) from the Open Knowledge Foundation
+  * [Free Cultural Works definition](https://freedomdefined.org/Definition) from Freedom Defined.org
+2. The license is structured to be generally usable by anyone. It is not a "vanity" license or specific to one project, consortium or corporation.
```

Comment:
```suggestion
2. The license is not specific to one project, consortium or corporation.
```

"Vanity" is pejorative, but not particularly clear. I associate it mostly with antiproliferation, in the sense of not welcoming new licenses with nearly the same purported legal effect as existing terms. But SPDX is full of licenses with substantially similar purported effects. And I've often found it useful precisely for that broader comprehensiveness. That allowed us to code [Blue Oak's permissive license list](https://blueoakcouncil.org/list) in SPDX IDs, for example.

Reply:
huh, never really thought of "vanity" as pejorative - just that it's a term used that seems to have general understanding among open source license folks (who've been around awhile) - but it's not a necessary term.

I do think that we should keep the "The license is structured to be generally usable by anyone" part (or the like) - I was trying to phrase things in the positive (then add the negative for color as needed). 

perhaps:
The license is structured to be generally usable by anyone. It is not specific to one project, consortium or corporation.

??

Reply:
Agree with the change in @jlovejoy's comment here -- important to keep "structured to be generally usable by anyone" but fine with the change to remove "vanity license."